app/controllers/help_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    gon.chat_enabled = AppConfig.chat.enabled?
```

Comment:
Doesn't work either. JS variable is `undefined` which make me think the variable is not corretly passed to the JS script.